Source/SourceKittenFramework/WindowsError.swift

```diff
@@ -0,0 +1,32 @@
+#if os(Windows)
+import WinSDK
+
+@_transparent
+internal func MAKELANGID(_ p: WORD, _ s: WORD) -> DWORD {
+    return DWORD((s << 10) | p)
+}
+
+struct WindowsError {
+    public let code: DWORD
```

Comment:
This declaration isn't actually public because its parent declaration is internal, leading to a SwiftLint violation.

```suggestion
    let code: DWORD
```